tailwind.config.js

```diff
@@ -0,0 +1,598 @@
+				},
+				'.cat-links,.tag-links': {
+					display: 'block',
+				},
+				'.post-edit-link': {
+					display: 'block',
+					marginBottom: config( 'theme.spacing.4' ),
+					marginTop: config( 'theme.spacing.4' ),
+				},
+				'.post-container': {
+					'&:not(:last-child)': {
+						marginBottom: config( 'theme.spacing.4' ),
+					},
+				},
+				'.post-navigation .nav-links': {
+					display: 'flex',
+					flexDirection: 'column',
+					textAlign: 'center',
+					'@screen tablet-landscape': {
+						flexDirection: 'row',
+						justifyContent: 'space-between',
+						textAlign: 'unset',
+					},
+				},
+				'.post-navigation .nav-next': {
+					marginTop: config( 'theme.spacing.4' ),
+					'@screen tablet-landscape': {
+						marginTop: '0',
+					},
+				},
+				'.comments-link': {
+					marginTop: config( 'theme.spacing.4' ),
+				},
+				'.comments-area': {
+					marginTop: config( 'theme.spacing.6' ),
+				},
+				'.comment-author': {
+					alignItems: 'center',
+					display: 'flex',
+					marginBottom: config( 'theme.spacing.4' ),
+				},
+				'.comment-author .avatar': {
+					marginRight: config( 'theme.spacing.4' ),
+				},
+				'.comment-author .fn': {
+					marginRight: config( 'theme.spacing.2' ),
+				},
+				'.comment-metadata': {
+					marginBottom: config( 'theme.spacing.4' ),
+				},
+				'.comment-body': {
+					padding: `${config( 'theme.spacing.2' )} 0`,
+				},
+				'.comment-content': {
+					backgroundColor: config( 'theme.colors.blue.100' ),
+					padding: config( 'theme.spacing.4' ),
+				},
+				'.comment-list .children': {
+					marginLeft: config( 'theme.spacing.4' ),
+				},
+				'.reply': {
+					paddingTop: config( 'theme.spacing.2' ),
+				},
+				'.comment-reply-title a': {
+					display: 'block',
+					fontSize: config( 'theme.fontSize.sm' ),
+				},
+				'.updated': {
+					'&:not(.published)': {
+						display: 'none',
+					},
+				},
+			}
+
+			const screenReaderText = {
+				'.screen-reader-text': {
+					clip: 'rect(1px, 1px, 1px, 1px)',
+					height: '1px',
+					overflow: 'hidden',
+					position: 'absolute',
+					whiteSpace: 'nowrap',
+					width: '1px',
+					'&:hover,&:active,&:focus': {
+						backgroundColor: config( 'theme.colors.blue.600' ),
+						clip: 'auto',
+						color: config( 'theme.colors.white' ),
+						display: 'block',
+						fontSize: config( 'theme.fontSize.base' ),
+						fontWeight: config( 'theme.fontWeight.medium' ),
+						height: 'auto',
+						left: '5px',
+						lineHeight: 'normal',
+						padding: config( 'theme.spacing.4' ),
+						textDecoration: 'none',
+						top: '5px',
+						width: 'auto',
+						zIndex: '100000',
+					},
+				},
+			}
+
+			const globalElements = {
+				'.button': {
+					padding: config( 'theme.spacing.4' ),
+				},
+				'*, ::after, ::before': {
+					boxSizing: 'border-box',
+					wordBreak: 'break-word',
+					wordWrap: 'break-word',
+				},
+				'pre': {
+					overflow: 'auto',
+				},
+			}
+
+			const menus = {
```

Comment:
Is the expectation to create a plugin to style blocks or other elements that are added? Like if I created a card, should I come in and create a tailwind plugin for cards?

I really don't like this CSS-in-JS syntax. Can we do this with `.css` or `.scss` files instead and use `@apply` to add tailwind utility classes? That way you have `header.php` for the markup and `header.scss` for the styles.

Reply:
The expectation right now is that the most base of base styles exist in the config, and styles specific to the theme/project would be added via Sass partials (or Tailwind classes in the markup, or whatever method is preferred). 

I'm not against moving these things out of the config file at all, though, if we'd rather have them live as regular Sass partials instead.